openmls/src/key_packages/test_key_packages.rs

```diff
@@ -15,38 +15,10 @@ fn generate_key_package(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoPr
     .expect("An unexpected error occurred.");
 
     // Generate a valid KeyPackage.
-    let lifetime_extension = Extension::Lifetime(LifetimeExtension::new(60));
-    let kpb = KeyPackageBundle::new(
-        &[ciphersuite],
-        &credential_bundle,
-        backend,
-        vec![lifetime_extension],
-    )
-    .expect("An unexpected error occurred.");
+    let kpb = KeyPackageBundle::new(&[ciphersuite], &credential_bundle, backend, vec![])
+        .expect("An unexpected error occurred.");
     std::thread::sleep(std::time::Duration::from_millis(1));
     assert!(kpb.key_package().verify(backend).is_ok());
-
-    // Now we add an invalid lifetime.
-    let lifetime_extension = Extension::Lifetime(LifetimeExtension::new(0));
-    let kpb = KeyPackageBundle::new(
-        &[ciphersuite],
-        &credential_bundle,
-        backend,
-        vec![lifetime_extension],
-    )
-    .expect("An unexpected error occurred.");
-    std::thread::sleep(std::time::Duration::from_millis(1));
-    assert!(kpb.key_package().verify(backend).is_err());
-
-    // Now with two lifetime extensions, the key package should be invalid.
-    let lifetime_extension = Extension::Lifetime(LifetimeExtension::new(60));
-    let kpb = KeyPackageBundle::new(
-        &[ciphersuite],
-        &credential_bundle,
-        backend,
-        vec![lifetime_extension.clone(), lifetime_extension],
-    );
-    assert!(kpb.is_err());
```

Comment:
As I understand it, there is no way to set a non-default lifetime in a KeyPackage for the time being, so we can't test this at the moment, although we definitely should. Do you plan to deal with this in a follow-up?

Reply:
Yes, setting lifetimes is all over the place right now. This will be cleaned up in one of the next PRs. I added it as one of the items to #819.

Reply:
Also note that key packages don't have lifetimes anymore but the leaf node has the lifetime now.